app/app/crashes/[record_locator]/page.tsx

```diff
@@ -8,7 +8,6 @@ import { GET_CRASH, UPDATE_CRASH } from "@/queries/crash";
 import { UPDATE_UNIT } from "@/queries/unit";
 import { UPDATE_PERSON } from "@/queries/person";
 import { useQuery } from "@/utils/graphql";
-import AppBreadCrumb from "@/components/AppBreadCrumb";
```

Comment:
the breadcrumb is now part of the outer layout and is going to be "on" by default everywhere. it has logic inside so that it doesn't render on top-level pages.

Reply:
awesome!